epoxy-adapter/build.gradle

```diff
@@ -1,26 +1,35 @@
 apply plugin: 'com.android.library'
+apply from: "$rootDir/blessedDeps.gradle"
```

Comment:
instead of doing `apply from: "$rootDir/blessedDeps.gradle"` in every module's `build.gradle` file can we apply it to all modules at once through the project level build.gradle?
```
subprojects { 
  apply from: "$rootDir/blessedDeps.gradle"
}
```

Not sure if that works or if there is right way to do it, but it would be a nice to have. Not needed to merge though